src/main/java/mate/academy/service/UserService.java

```diff
@@ -15,6 +15,14 @@ public class UserService {
      * Return <code>null</code> if there is no suitable user
      */
     public User findByEmail(String email) {
+        if (email == null) {
+            return null;
+        }
```

Comment:
Is this check really necessary? What will be returned if email = null without this check?